Tests/GeoLib/TestLineSegmentIntersect2d.cpp

```diff
@@ -50,18 +50,29 @@ class LineSegmentIntersect2dTest : public testing::Test
 	ac::gtest_reporter gtest_reporter;
 };
 
+#if not defined(_MSC_VER) && _MSC_VER < 1900
+// Compilers of MVS below 2015 do not support unrestricted unions. The
+// unrestricted union is used by autocheck to handle test data. The autocheck
+// workaround for MVS compilers (below version 2015) contains a bug and in the
+// consequence the tests crashes. For this reason the tests are disabled under
+// this environments.
```

Comment:
> What is the advantage to "#if ... #else ... #endif" each particular test instead of all tests at once?

When doing so, the word "DISABLED_" occurs explicitly in the source code. So it is rather easy to get all disabled tests. Furthermore the number of disabled tests is reported by testrunner. Furthermore it is at least made sure that the test itself can be compiled, even though it might crash at runtime. So as long as it compiles, I'd prefer disabling tests one by one.